envi/memory.py

```diff
@@ -431,6 +431,36 @@ def addMemoryMap(self, va, perms, fname, bytez):
         self._map_defs.append(hlpr)
         return
 
+    def delMemoryMap(self, va):
+        '''
+        Delete a memory map from this object...
```

Comment:
can we document that the `va` parameter is the map starting address? otherwise, its not clear from the signature if this is the starting address or an address contained within a map.

Reply:
good call.  i'll rename it `mapva`